src/components/admin/settings-tab.yml

```diff
@@ -40,6 +40,12 @@ templateDocumentHeader:
 licenseHeader:
   en: License
   de: Lizenz
+dataMigrationHeader:
+  en: Data migration
+  de: Datenmigration
 lastExecution:
   en: Last execution
   de: Letzte Ausführung
+pluginsHelpTextsHeader:
+  en: Plugins help texts
+  de: Plugins-Hilfetexte
```

Comment:
Maybe "Hilfetexte für Plugins" sounds a bit friendlier and less bureaucratic (similar to "Vorlage für neue Dokumente"), in English I would also find "Help texts for plugins" nicer.